projects/packages/forms/tests/php/contact-form/Contact_Form_Test.php

```diff
@@ -1460,7 +1460,8 @@ public function test_make_sure_checkbox_multiple_field_renders_as_expected() {
 						'class' => 'has-text-color',
 						'style' => 'color:gummy; font-size:14px;',
 					),
-				)
+				),
+				JSON_UNESCAPED_SLASHES
```

Comment:
This will probably need changing to match my suggested change above to the code being tested.

```suggestion
				JSON_UNESCAPED_SLASHES | JSON_HEX_AMP
```